typings/index.d.ts

```diff
@@ -41,6 +41,10 @@ declare module 'discord-giveaways' {
 
         public emit<K extends keyof GiveawaysManagerEvents>(event: K, ...args: GiveawaysManagerEvents[K]): boolean;
     }
+    interface BonusEntry {
+        filter (member: GuildMember): boolean|Promise<boolean>;
+        bonus: number;
```

Comment:
So here (and everywhere else for logic), it would become:
```suggestion
        bonus (member: GuildMember): number|Promise<number>;
        cumulative: boolean;
```